pkg/test/auth/tokenmanager_test.go

```diff
@@ -0,0 +1,173 @@
+package test
```

Comment:
package should be `auth` (or even `auth_test` if possible)

Reply:
fixed: 15dc7702d0869b27c7bd3c0a2bf96e2166c0e7f6